config/vufind/config.ini

```diff
@@ -2156,13 +2156,21 @@ includeSchemaOrgMetadata = true
 
 ; The following two sections control the Alphabetic Browse module.
 [AlphaBrowse]
+
```

Comment:
We don't usually have blanks at the start of sections; might be worth removing this one for consistency.

Reply:
Sorry, nervous habit :)